docs/ldml/tr35-personNames.md

```diff
@@ -346,9 +344,9 @@ The order attribute is used for patterns with different orders of fields. The or
 
 For example, when the display language is Japanese, it is customary to use _surnameFirst_ for names of people from Japan and Hungary, but use _givenFirst_ for names of people from the United States and France. Although the English pattern for sorting is distinct from the other patterns (except for unusual names), that is not necessarily the case in other languages.
 
-### 4.2 <a name="4-2-length" href="#4-2-length">length</a>
+### 4.2 <a name="length" href="#length">length</a>
 
-The `length` attribute specifies the relative length of a formatted name depending on context. For example, a `long` formal name in English would include prefix, given, given2, surname plus suffix; whereas a `short` informal name may only be the given name.
+The `length` attribute specifies the relative length of a formatted name depending on context. For example, a `long` formal name in English might include title, given, given2, surname plus generation and credentials; whereas a `short` informal name may only be the given name.
```

Comment:
Would this read better if you had the formality in the code typeface too (i.e., `long formal` and `short informal`)?

Reply:
good idea. Might be better to do this in a follow-on PR. (I think it was a mistake to fix the malformed anchors in this PR because it makes it harder to see the material changes)